yt/yt/ytlib/query_tracker_client/helpers.cpp

```diff
@@ -14,17 +14,26 @@ using namespace NYson;
 
 TString GetFilterFactors(const TActiveQueryPartial& record)
 {
-    return Format("%v %v", record.Query, *record.Annotations ? ConvertToYsonString(*record.Annotations, EYsonFormat::Text).ToString() : "");
+    return Format("%v %v aco:%v",
+        record.Query,
+        record.Annotations && *record.Annotations ? ConvertToYsonString(*record.Annotations, EYsonFormat::Text).ToString() : "",
```

Comment:
`record.Annotations && *record.Annotations` is better to round with parenthesis (to simplify reading this code)

And fix it also below, please.